fitbenchmarking/fitbenchmarking
# 361 error handling

#### Description of Work

Fixes #361 

Adds new error flag and message to controllers.

#### Testing Instructions

1. Check flags are set correctly

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
